tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, unit, integration
+envlist = lint, static, unit, integration
```

Comment:
I would remove integration from this list. Integration tests generally take a really long time and (this is of course my opinion) running plain tox should be quick enough that I can do it often while developing.

Reply:
I have a pre-push global hook that runs lint and unit explicitly.
I think the semantics of "tox without args" should be suitable for running a monolithic github workflow, i.e. test everything.

Reply:
I don't really see the logic here. With a github workflow, you put whatever cli args you want once and it is saved forever. Compare this to having to run `tox -e lint,unit` every time you want to run the tests. I would much rather have to use the args in the github action.

Reply:
That is a valid point indeed.
However there is only one meaning we can attribute to "tox without args", and different devs could have different ideas about what makes sense to check every time. For example would everyone run static analysis? So for consistency it is probably much easier to decide (if at all needs to be decided) that "tox without args" = everything = no ambiguity across devs.